apps/reaper/lib/reaper/xml/xml_stream/sax_handler.ex

```diff
@@ -0,0 +1,133 @@
+defmodule XMLStream.SaxHandler do
+  @moduledoc false
+
+  defmodule State do
+    @moduledoc false
+    @enforce_keys [:tag_path, :emitter]
+    defstruct stack: [], accumulate: false, tag_stack: [], tag_path: [], emitter: nil
+
+    def new(fields \\ %{}) do
+      struct!(__MODULE__, fields)
+    end
+
+    def update(%__MODULE__{}=state, fields) do
+      struct!(state, fields)
+    end
+  end
+
+  @behaviour Saxy.Handler
+  @file_chunk_size 40_000
```

Comment:
Should this be configurable?

Reply:
Not anymore than it is as a module attribute. If we ever discover a scenario where it should be we can make it configurable then.